### PR TITLE
Add build for browser

### PR DESCRIPTION
**Pull request by nechaido**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,12 +21,16 @@
   "scripts": {
     "test": "eslint . && node test/all.js",
     "lint": "eslint .",
-    "docs": "metaschema ./common.js"
+    "docs": "metaschema ./common.js",
+    "build": "babel common.js -d dist && babel lib -d dist/lib",
+    "prepublish": "npm run -s build"
   },
```
Comment (aqrln) [inline package.json]:
```json
  "browser": {
    "common.js": "./dist/common.js"
  },
```

Comment (nechaido) [reply package.json]: Fixed.

**Comment by nechaido** [comment]
@aqrln ping.

**Comment by nechaido** [comment]
Landed in  5e3b0ef.

**Comment by tshemsedinov** [comment]
@nechaido if we don't need branch after merge/land, please delete

**Pull request closed without merging**